packages/lesswrong/components/ea-forum/giving-portal/EAGivingPortalPage.tsx

```diff
@@ -0,0 +1,440 @@
+    lineHeight: "150%",
+    "& a": {
+      color: "inherit",
+      textDecoration: "underline",
+      fontWeight: 700,
+    },
+  },
+  button: {
+    display: "flex",
+    alignItems: "center",
+    gap: "10px",
+    fontSize: 16,
+    fontWeight: 600,
+    background: theme.palette.grey[0],
+    color: theme.palette.givingPortal[1000],
+    borderRadius: theme.borderRadius.small,
+    padding: "12px 16px",
+    border: "none",
+    outline: "none",
+    "&:hover": {
+      opacity: 0.85,
+    },
+    "&:active": {
+      opacity: 0.7,
+    },
+  },
+  postsList: {
+    width: SECTION_WIDTH,
+    maxWidth: "100%",
+    "& .LoadMore-root": {
+      color: theme.palette.grey[600],
+    },
+  },
+  primaryLoadMore: {
+    "& .LoadMore-root": {
+      color: theme.palette.givingPortal[1000],
+    },
+  },
+  electionCandidates: {
+    width: 1120,
+    maxWidth: "100%",
+    display: "flex",
+    flexWrap: "wrap",
+    gap: "16px",
+    rowGap: "12px",
+  },
+  donationOpportunities: {
+    width: "100%",
+    display: "flex",
+    flexWrap: "wrap",
+    gap: "16px",
+    rowGap: "16px",
+    justifyContent: "center",
+  },
+  totalRaised: {
+    fontSize: 24,
+    fontWeight: 700,
+    letterSpacing: "-0.24px",
+    lineHeight: "140%",
+    paddingLeft: 4,
+  },
+  loadMore: {
+    color: theme.palette.givingPortal[1000],
+    fontSize: 16,
+    fontWeight: 700,
+    letterSpacing: "-0.16px",
+    cursor: "pointer",
+    "&:hover": {
+      opacity: 0.8,
+    },
+  },
+  mt10: { marginTop: 10 },
+  mt20: { marginTop: 20 },
+  mt30: { marginTop: 30 },
+  mt60: { marginTop: 60 },
+  mt80: { marginTop: 80 },
+  mb40: { marginBottom: 40 },
+  mb80: { marginBottom: 80 },
+  mb100: { marginBottom: 100 },
+});
+
+const useAmountRaised = () => {
+  // TODO: Query for the actual amount
+  return {
+    raisedForElectionFund: 3720,
+    donationTarget: 15000,
+    totalRaised: 10250,
+  };
+}
+
+const donationElectionLink = "#"; // TODO
+
+const votingOpensDate = new Date("2023-12-01");
+
+const donationElectionTagId = "L6NqHZkLc4xZ7YtDr"; // TODO: This tag doesn't exist yet
+const effectiveGivingTagId = "L6NqHZkLc4xZ7YtDr";
+
+const timelineSpec: TimelineSpec = {
+  start: new Date("2023-11-15"),
+  end: new Date("2023-12-31"),
+  points: [
+    {date: new Date("2023-11-28"), description: "Giving Tuesday"},
+    {date: votingOpensDate, description: "Voting starts"},
+    {date: new Date("2023-12-15"), description: "Voting ends"},
+    {date: new Date("2023-12-20"), description: "Election winner announced"},
+  ],
+  spans: [
+    {
+      start: new Date("2023-11-21"),
+      end: new Date("2023-11-28"),
+      description: "Effective giving spotlight Week",
+    },
+    {
+      start: new Date("2023-11-30"),
+      end: new Date("2023-12-07"),
+      description: "Marginal Funding Week",
+    },
+    {
+      start: new Date("2023-12-08"),
+      end: new Date("2023-12-16"),
+      description: "Forum BOTEC-a-thon Week",
+    },
+  ],
+};
```

Comment:
These timeline dates are off by one for me:

<img width="1139" alt="Screen Shot 2023-10-23 at 6 08 24 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/d3794568-9b0c-4aeb-808a-1bf27ca2ef11">